docs/infrastructure/administration.md

```diff
@@ -0,0 +1,416 @@
+# Infrastructure administration
+
+## Obtaining access
+
+- Heroku: Follow the [Mozilla Heroku SSO guide] then ask to be invited to the Heroku apps.
+- Amazon RDS: File a [Treeherder infrastructure bug] requesting access to the
+  `moz-devservices` AWS account and needinfo `:dividehex`, who will update the
+  [IAM configuration file][iam-config].
+- New Relic: Ask for an invite to be sent using the [New Relic user management] page.
+- Papertrail: Ask for an invite to be sent using [Papertrail user management] page.
+
+[mozilla heroku sso guide]: https://mana.mozilla.org/wiki/display/TS/Using+SSO+with+your+Heroku+account
+[treeherder infrastructure bug]: https://bugzilla.mozilla.org/enter_bug.cgi?product=Tree%20Management&component=Treeherder%3A%20Infrastructure
+[new relic user management]: https://account.newrelic.com/accounts/677903/users
+[papertrail user management]: https://papertrailapp.com/account/members
```

Comment:
This link redirects me to https://papertrailapp.com/account/profile
And I get this message: "Membership in Heroku add-on accounts is handled by Heroku. Check out our Heroku support page for more information."

This is probably because I have Papertrail set up with the Heroku app 'firefox-health-backend' as an add-on.

Reply:
Yeah the UX for mixing Heroku-addon and standalone account Papertrail logins isn't great. I've emailed their support in the past about it, but there's not much we can do - though thankfully this affects very few people.

Note that Taskcluster (whose Papertrail account we share) is possible moving to another logging solution, at which point Treeherder would either follow, or else switch to the Papertrail addon (making the UX issue, and the whole need to request separate access to standalone Papertrail unnecessary).